terraform/modules/aws-cargohold-asset-storage/README.md

```diff
@@ -0,0 +1,17 @@
+Terraform module: Cargohold Asset Storage
+=========================================
+
+This module creates an Object Storage on AWS for cargohold to store encrypted assets.
+
+
+#### How to use the module
+
+```hcl-terraform
```

Comment:
`hcl-terraform` is not a valid syntax. Use either `terraform` or `hcl`, which [should be supported](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml), but `hcl-terraform` is not.